src/shellstrings.c

```diff
@@ -7,88 +7,331 @@
-    char *i = "has been accepted as the input file for";
-    printf("%s %s %s!\n", input_file, i, status);
+void shell_help(bool *color) {
+    if (*color == true) {
+        printf(BOLDWHITE "------- Help Page ------\n" RESET
+
+               "\nSpellcheck is a tool which searches for misspelled words "
+               "and suggests alternative spellings given a text file.\n"
+               
+               "\nCurrently, you are in interactive mode. Return to the previous page to input a file.\n"
+               
+               "\nTo run in batch mode, exit the program with the command " BOLDWHITE "exit" RESET
+               ", and run: " BOLDWHITE "$ ./spellcheck -i /path/file.txt" RESET ".\n"
+               
+               "\nFlags: " BOLDWHITE "-q" RESET " is for quiet, and " BOLDWHITE
+               "-v" RESET " is for verbose.\n"
+               
+               "\nTo input a custom dictionary to the program, use " BOLDWHITE 
+               "-d /path/dictionary.txt" RESET ", and to provide a file saving destination, "
+               "use " BOLDWHITE  "-s /path/filename.txt" RESET ".\n"
+               
+               "\nTo enable color inside the program, use the " BOLDWHITE "-c" RESET " flag.\n"
+
+               BOLDWHITE "\nEnter any command to return to the previous page." RESET "\n");
+    }
+
+    else {
+        printf("------- Help Page ------\n"
+
+               "\nSpellcheck is a tool which searches for misspelled words "
+               "and suggests alternative spellings given a text file.\n"
+               
+               "\nCurrently, you are in interactive mode. Return to the previous page to input a file.\n"
+               
+               "\nTo run in batch mode, exit the program with the command 'exit', "
+               "and run: '$ ./spellcheck [-flag] /path/file.txt'.\n"
+               
+               "\nFlags: '-q' is for quiet, and '-v' is for verbose.\n"
+               
+               "\nTo input a custom dictionary to the program, use " 
+               "'-d /path/dictionary.txt', and to provide a file saving destination, "
+               "use '-s /path/filename.txt'.\n"
+               
+               "\nTo enable color inside the program, use the '-c' flag.\n"
+
+               "\n-- Enter any command to return to the previous page. --\n");
+    }
 }
 
-/* See shellstrings.h */
-void shell_help() {
-    printf("Spellcheck is a tool which searches for misspelled words and suggests alternative spellings given a text file.\n");
-    printf("For interactive mode, run: r [/path/file.txt]\n");
-    printf("For batch mode, exit the program, and run: $ ./spellcheck [/path/file.txt] [-flag]\n");
-    printf("Flags:  -q is for quiet, -v is for verbose, and -s is for saving a file to a custom destination.\n");
-    printf("Enter any command to return to the previous page.\n");
+void shell_start_interactive(char *filename, char *dict, char *md, bool *color) {
```

Comment:
Why is this a `bool *` and not just a `bool`?

Reply:
changed!